app/src/main/java/us/shandian/giga/io/StoredFileHelper.java

```diff
@@ -37,6 +41,19 @@
     private String srcName;
     private String srcType;
 
+    public StoredFileHelper(final Context context, final Uri uri, final String mime) {
```

Comment:
Perhaps we should move `StoredFileHelper` and `StoredDirectoryHelper` to NewPipe as well.

Reply:
See #3781